engine/runtime/src/main/scala/org/enso/compiler/phase/ExportsResolution.scala

```diff
@@ -205,6 +213,14 @@ class ExportsResolution {
       ).flatten.groupBy(_._1).map { case (m, names) =>
         (m, names.flatMap(_._2).distinct)
       }
+
+      bindings.exportedSymbols = bindings.allExportedSymbols.flatMap {
```

Comment:
This is a second time `exportedSymbols` are computed as `allExportedSymbols.flatMap`...

I am no Scala guru, but shouldn't this be achieved via Scala's _lazy value_? Something like:
```scala
class ExportedSymbols(allExportedSymbols) {
  lazy val exportedSymbols = allExportedSymbols.flatMap(...)
}
```

Reply:
`allExportedSymbols` is a var, so no